xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-icon/plugin.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+(function () {
+  'use strict';
+
+  CKEDITOR.plugins.add('xwiki-icon', {
+    init: function (editor) {
+      editor.config.mentions = editor.config.mentions || [];
+      // Icon Autocomplete
+      editor.config.mentions.push({
+        marker: `/icon::`,
```

Comment:
I removed the conflict [when introducing the image insertion quick action](https://github.com/xwiki/xwiki-platform/commit/1eed1c849aa852824e8e3f4961fcc472e4e8992d#diff-4060dab34e60de96a5790b5872408cc687019aab0dd4ebb874121c5215c7f096R350) (similar `/img::`). My idea was that it would make it part of Quick Actions and not crowd the shortcut space.

Reply:
Yes, but I don't see the benefit. It's still an additional shortcut, and it's complex enough without the slash prefix so that it doesn't get triggered accidentally. What if I really want to search for "img::" in quick actions? Quick actions are extensible, so we can't know what use cases there might be in the future that might require a search for "img::". I don't think it's right to prevent this search.